js/index.js

```diff
@@ -1 +1,100 @@
 // Your code goes here
+const nav = document.querySelector(".nav-container");
+
+nav.addEventListener("click", (event) => {
+    event.target.style.backgroundColor = 'blue';
+});
+
+const navLinks = document.querySelectorAll(".nav a");
+
+for (let i = 0; i < navLinks.length; i++) {
+    navLinks[i].addEventListener("mouseover", (event) => {
+        event.target.style.color = 'Gainsboro';
+    });
+
+    navLinks[i].addEventListener("mouseout", (event) => {
+        event.target.style.color = '#212529';
+    });
+
+    navLinks[i].addEventListener("click", (event) => {
+        event.stopPropagation();
+        event.preventDefault();
+        event.target.style.fontSize = 'large';
+    });
+}
```

Comment:
NodeLists have a forEach method you can use!

https://developer.mozilla.org/en-US/docs/Web/API/NodeList